benchmarks/benchmarks/bench_manipulate.py

```diff
@@ -0,0 +1,104 @@
+from .common import Benchmark, get_squares_, TYPES1, DLPACK_TYPES
+
+import numpy as np
+from collections import deque
+
+class BroadcastArrays(Benchmark):
+    params = [[(16, 32), (32, 64),
+               (64, 128), (128, 256),
+               (256, 512), (512, 1024)],
+              TYPES1]
+    param_names = ['shape', 'ndtype']
+    timeout = 10
+
+    def setup(self, shape, ndtype):
+        self.xarg = np.random.ranf(shape[0]*shape[1]).reshape(shape)
+        self.xarg = self.xarg.astype(ndtype)
+        if ndtype.startswith('complex'):
+            self.xarg += np.random.ranf(1)*1j
+
+    def time_broadcast_arrays(self, shape, ndtype):
+        np.broadcast_arrays(self.xarg, np.ones(1))
+
+class BroadcastArraysTo(Benchmark):
+    params = [[16, 32, 64, 128, 256, 512],
+              TYPES1]
+    param_names = ['size', 'ndtype']
+    timeout = 10
+
+    def setup(self, size, ndtype):
+        self.xarg = np.random.ranf(size)
+        self.xarg = self.xarg.astype(ndtype)
+        if ndtype.startswith('complex'):
+            self.xarg += np.random.ranf(1)*1j
```

Comment:
Use `np.random.random`.  Since you are not seeding, actually even nicer `rng = np.default_rng()` and `rng.uniform()`.

Reply:
Fixed, though maybe the benchmarks should have a seed set..